### ISSUE TITLE
Failed to commit transaction (conflicting files) for VS Code and Teams for Linux

### ISSUE THREAD

**Issue by guru-das-s** [Jguer/yay#2487]
### Affected Version

```
$ yay -V
yay v12.3.5 - libalpm v14.0.0
```

### Reproduction Steps

1. Just use `yay -Syu`

### Expected behavior

Building from source and installation of all packages installed using `yay`

### Output

```sh
==> Sources are ready.
loading packages...
resolving dependencies...
looking for conflicting packages...

Packages (6) google-chrome-127.0.6533.99-1  teams-for-linux-1.8.1-1  teams-for-linux-debug-1.8.1-1  visual-studio-code-bin-1.92.1-1  visual-studio-code-bin-debug-1.92.1-1  zoom-6.1.6-1

Total Installed Size:  1660.43 MiB
Net Upgrade Size:         8.60 MiB

:: Proceed with installation? [Y/n] y
(6/6) checking keys in keyring                                                                                     [####################################################################] 100%
(6/6) checking package integrity                                                                                   [####################################################################] 100%
(6/6) loading package files                                                                                        [####################################################################] 100%
(6/6) checking for file conflicts                                                                                  [####################################################################] 100%
error: failed to commit transaction (conflicting files)
/usr/lib/debug/.build-id/04/e9960947a38ced0a87421a6f198b9caf43dc3f exists in both 'visual-studio-code-bin-debug' and 'teams-for-linux-debug'
/usr/lib/debug/.build-id/04/e9960947a38ced0a87421a6f198b9caf43dc3f.debug exists in both 'visual-studio-code-bin-debug' and 'teams-for-linux-debug'
/usr/lib/debug/.build-id/37/851ad3790029c78d20ee4b4761055d5ad5dab6 exists in both 'visual-studio-code-bin-debug' and 'teams-for-linux-debug'
/usr/lib/debug/.build-id/37/851ad3790029c78d20ee4b4761055d5ad5dab6.debug exists in both 'visual-studio-code-bin-debug' and 'teams-for-linux-debug'
/usr/lib/debug/.build-id/68/3278eba471e7069eeddf5a8270d9a201ea88da exists in both 'visual-studio-code-bin-debug' and 'teams-for-linux-debug'
/usr/lib/debug/.build-id/68/3278eba471e7069eeddf5a8270d9a201ea88da.debug exists in both 'visual-studio-code-bin-debug' and 'teams-for-linux-debug'
Errors occurred, no packages were upgraded.
 -> error installing: [/home/gurus/.cache/yay/zoom/zoom-6.1.6-1-x86_64.pkg.tar.zst /home/gurus/.cache/yay/visual-studio-code-bin/visual-studio-code-bin-1.92.1-1-x86_64.pkg.tar.zst /home/gurus/.cache/yay/visual-studio-code-bin/visual-studio-code-bin-debug-1.92.1-1-x86_64.pkg.tar.zst /home/gurus/.cache/yay/google-chrome/google-chrome-127.0.6533.99-1-x86_64.pkg.tar.zst /home/gurus/.cache/yay/teams-for-linux/teams-for-linux-1.8.1-1-x86_64.pkg.tar.zst /home/gurus/.cache/yay/teams-for-linux/teams-for-linux-debug-1.8.1-1-x86_64.pkg.tar.zst] - exit status 1
```

**Comment by eduarddejong** [comment]
Having the same trouble, but completely outside of Yay.
I reported it here: https://aur.archlinux.org/packages/vscodium-bin not yet being aware of the scale of the issue.

Turns out that Arch Linux itself made some change so that debug packages are now a thing by default.

Personally I am going to follow the suggestions I could found to disable debug packages by default in makepkg.conf by adding an exclamation mark in front of it:
https://bbs.archlinux.org/viewtopic.php?id=293005
https://www.reddit.com/r/archlinux/

However this is in fact still a work around, packages shouldn't conflict in my opinion.

I am curious if Yay or any other AUR helper would even be able to actually "fix" it (other than just recommending the user to disable debug packages or give the user some option to automatically do it).
